.coafile

```diff
@@ -47,3 +47,14 @@ ignore = vendor/**, resources/vendors/**, _site/**
 bears = YAMLLintBear
 files = **.yml
 ignore = vendor/**
+
+[projects_ignore]
+files = _projects/**.md
+bears = LineLengthBear
+max_line_length = 80
+ignore = _projects/corobo_enhancement.md,_projects/bear_updates.md,_projects/README.md,_projects/improve_installation.md,_projects/commit_based_perftest.md,_projects/automate_dependency_repology.md,_projects/automate_vulture_whitelist.md,_projects/integrate-pyflakes-AST.md,_projects/rstcheck_with_better_sphinx_support.md,_projects/nextgen_action_system.md,_projects/implement_aspects.md,_projects/debug_profile.md,_projects/improve_diff_handling.md,_projects/jetbrains-ide-plugin.md,_projects/cobot_enhancement.md,_projects/example.md,_projects/aspects.md,_projects/communication_bridge.md,_projects/assign_issues.md,_projects/coala_web.md,_projects/use_coala.md,_projects/enhance_coala_quickstart.md,_projects/coala_cli.md,_projects/go_bears_on_bsd.md,_projects/gitmate_gitlab.md,_projects/optimize_caching.md,_projects/documentation_extraction.md,_projects/git_task_list.md,_projects/gitmate_jira.md,_projects/use_coala_2.md,_projects/gitmate_plugins.md,_projects/use_coala_3.md,_projects/vulture.md,_projects/create_jenkins_plugin.md,_projects/implement_metrics_for_coala.md,_projects/newcomer_metrics.md,_projects/editor_support.md,_projects/extend_linter_integration.md,_projects/pr_perftest.md,_projects/add_repo_license.md,_projects/antlr_v4.md,_projects/coala_language_server.md,_projects/gitmate_testing.md,_projects/quickstart_green_mode.md,_projects/commit-content-checks.md,_projects/nested_languages.md,_projects/meta_review.md,_projects/convert_bear_to_aspects.md,_projects/roberta_external_source.md,_projects/generic_bears.md
```

Comment:
You can do 

```
ignore = _projects/README.md,
  _projects/bear_updates.md,
  etc
  last_file_without_comma

```

Reply:
I don't know if re-structuring existing files is allowed or not,
But if it is, I suggest to create a sub-folder called old_projects and put all these files under that sub-folder

Then this ugly list of folders can be removed by using a glob pattern `_projects/old_projects/**.md`

(PS: This is a jekyll feature that fishes out posts from sub-directories automatically, and it treats the sub-folders in the exact same way, especially for this repo, since we use `project.name` in order to determine the link for a project)

Reply:
IMO switching files to a different folder just for listing purposes doesn't seem like a great idea. To a newcomer it would sound preposterous that we made a different folder just so that it was easier to apply `max_line_length`